app/lib/dartdoc/customization.dart

```diff
@@ -75,6 +75,7 @@ class DartdocCustomizer {
     _addAlternateUrl(head, canonical);
     _addAnalyticsTracker(head, body);
     _addGithubMarkdownStyle(head, body);
+    _replaceFontUrls(head);
```

Comment:
Why is this needed?
https://github.com/dart-lang/dartdoc/blob/master/lib/templates/html/_head.html#L25

Is it just we want it for material icons too?